src/Types.ts

```diff
@@ -706,3 +709,15 @@ export interface SubmitStrategyRequest {
 export interface SubmitStrategyResponse {
   hash: string
 }
+
+export interface requireBitUsedRequest {
+  bit: BitJSON;
```

Comment:
@mikec or @MatiasOS when defining a primitiveArgs. should it nest this inside a `bit` key?

Reply:
@phanpak I followed the pattern we're using for the UseBit primitive and used index/value. We're trying to keep all user facing naming to index/value. See https://github.com/brinktrade/brink-sdk/commit/65c1f2723a631cc6d36379034c638256bfc51a28#diff-02e99e848089f1a0a30aa7b26396bfd88f111a0d5914da4929901d3281e5cd98R21

For types repo, we don't need to add these request/response types here (see above)